### PR TITLE
Add doc example for `std::time::Instant::elapsed`.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: src/libstd/time/mod.rs [hunk 1]
```diff
@@ -150,6 +150,18 @@ impl Instant {
     /// This function may panic if the current time is earlier than this
     /// instant, which is something that can happen if an `Instant` is
     /// produced synthetically.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
```
Comment (GuillaumeGomez) [inline src/libstd/time/mod.rs]: You can remove `rust`, it's the default value.

**Comment by GuillaumeGomez** [comment]
Just one thing and it's good to go!

**Comment by frewsxcv** [comment]
Done

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**